config/v1/types_network.go

```diff
@@ -85,6 +86,16 @@ type NetworkStatus struct {
 
 	// Migration contains the cluster network migration configuration.
 	Migration *NetworkMigration `json:"migration,omitempty"`
+
+	// conditions represents the observations of a network.config current state.
+	// Known .status.conditions.type are: "LiveMigrationRunning", "LiveMigrationRoutableMTUApplied",
```

Comment:
Could you update the possible types according to https://github.com/openshift/enhancements/blob/76ea48a88bca15ef6157aacf90b8544a763e6075/enhancements/network/sdn-live-migration.md?plain=1#L352?

Reply:
Done